1/homework/README.txt

```diff
@@ -0,0 +1,12 @@
+��� ������ ��� ��������������� ���������� �������� ������ � ������� WAV ��� FLAC.
```

Comment:
Bad file encoding for github
Also I'd prefer .md files instead of .txt for README